### PR TITLE
Add detailed pull request guidelines

### PR DESCRIPTION
**Pull request by amulware**
It is rather extensive, but the details seems necessary to me. Comments?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Requirements:
 - [XCode](https://developer.apple.com/download/)
 - [CocoaPods](https://guides.cocoapods.org/using/getting-started.html#installation)
 
-Downloading and starting development for superday is supersimple®:
+Downloading and starting development for Superday is supersimple™:
```
Comment (willsb) [inline README.md]: This triggers me, as I like ® better, but ™ is indeed more correct :(

File: README.md [hunk 2]
```diff
@@ -24,7 +24,41 @@ Downloading and starting development for superday is supersimple®:
...
+    - Change as many files as necessary, but as few as possible
+    - Do not add unrelated changes - even minor code cleanup - to your pull request
+        - Create a separate pull request instead
+        - If possible, split larger pull requests into multiple ones
+3. Pull requests should be presented well
+    - Use a clear and succinct title
+        - If this is not possible, the pull request is most likely too big
+    - Add a description of needed
+    - Add references to related issues/pull requests to the description
+        - If the pull request closes an issue, include `Closes #issue`
```
Comment (willsb) [inline README.md]: It'd be nice to have a link to [this page](https://github.com/blog/1386-closing-issues-via-commit-messages) here

Comment (amulware) [inline README.md]: Actually, [this one](https://github.com/blog/1506-closing-issues-via-pull-requests).

**Comment by amulware** [comment]
Find me a reasonable acronym and we'll add a tag. Until then, feel free to update/squash then when the tests are through.

**Pull request merged**